user_docs/en/userGuide.md

```diff
@@ -3387,6 +3387,14 @@ It is very important to only install add-ons from sources you trust.
 The functionality of add-ons is unrestricted inside NVDA.
 This could include accessing your personal data or even the entire system.
 

```

Comment:
```suggestion
Add-ons are scanned by [Virus Total](https://www.virustotal.com/), however, this only detects known malware at submission time.

```

Reply:
I think it would be better to part of the previous paragraph

Reply:
I initially had it at the end of that paragraph, but just reading it again as a whole:

"Just because an add-on is available in the NVDA Add-on Store, does not mean that it has been approved or vetted by NV Access or anyone else. It is very important to only install add-ons from sources you trust. The functionality of add-ons is unrestricted inside NVDA. This could include accessing your personal data or even the entire system."

Perhaps between "... approved or vetted by NV Access or anyone else." and "It is very important to only install add-ons from sources you trust..." would be better?

Reply:
that makes sense to me